tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/ElasticsearchTests.java

```diff
@@ -80,20 +82,20 @@
      * 
      * Site1:
      *  - Users: userSite1, userMultiSite
-     *  - Documents: FILE_0_NAME (owner: userSite2), FILE_1_NAME (owner: userSite2), FILE_3_NAME (owner: userSite1)
+     *  - Documents: FILE_0_NAME (owner: userSite1), FILE_1_NAME (owner: userSite1), FILE_3_NAME (owner: userSite2)
      *  
      * Site2:
      *  - Users: userSite2, userMultiSite
      *  - Documents: FILE_2_NAME (owner: userSite2)
      *  
      */
     @BeforeClass(alwaysRun = true)
-    public void dataPreparation()
+    public void dataPreparation() throws IOException
     {
         serverHealth.assertServerIsOnline();
 
         elasticClient = new RestHighLevelClient(RestClient.builder(new HttpHost("localhost", 9200, "http")));
```

Comment:
The connection parameters should not be hardcoded.

Reply:
This line doesn't exist anymore, it is removed after the Elasticsearch client removal.